src/VerifyWindowsHelloController.js

```diff
@@ -54,6 +59,17 @@ function (Okta, FormController, FormType, webauthn, FooterSignout, WindowsHelloE
                 clientData: assertion.clientData,
                 signatureData: assertion.signature
               });
+            })
+            .fail(function (error) {
+              switch (error.message) {
+              case 'AbortError':
+              case 'NotFoundError':
+              case 'NotSupportedError':
+                model.trigger('abort', error.message);
+                return transaction;
```

Comment:
Sure, we can.
I have renamed the variable `factorTransaction` to `verifyData` and it should become more understandable.

`factorTransaction` was a request result of the